packages/testkit-backend/src/console.remote.js

```diff
@@ -0,0 +1,31 @@
+import { response } from './responses'
+
+const originalConsole = console
+
+export default {
+  install: (channel) => {
+    // eslint-disable-next-line no-global-assign
+    console = new Proxy({}, {
+      get: (_, method) => (...args) => {
+        originalConsole[method].apply(originalConsole, args)
+        channel.writeResponse(null, response('Console', {
+          method,
+          args
+        }), true)
+      }
+    })
+  },
+  handleConsole: (message) => {
+    if (message.response.name === 'Console') {
+      const { method, args } = message.response.data
+      args[0] = typeof args[0] === 'string' ? `[RemoteConsole] ${args[0]}` : args[0]
+      console[method].apply(console, args)
+      return true
+    }
+    return false
+  },
+  uninstall: () => {
```

Comment:
More of a question - do we need/use this uninstall function or is this just for future-proofing?

Reply:
We don't need, this is only implemented for future-proofing and for keeping the symmetry.